src/assets/scripts/client/parsers/aircraftCommandParser/aircraftCommandMap.js

```diff
@@ -1,40 +1,6 @@
-/**
- * List of System Commands
- *
- * When a command is parsed, the value here will be used for the `name` property
- * of the `AircraftCommandParser`
- *
- * @property SYSTEM_COMMANDS
- * @type {Object}
- * @final
- */
-export const SYSTEM_COMMANDS = {
-    auto: 'auto',
-    clear: 'clear',
-    pause: 'pause',
-    tutorial: 'tutorial',
-
-    // single arg commands
-    '`': 'moveDataBlock',
-    airport: 'airport',
-    rate: 'rate',
-    timewarp: 'timewarp',
-    transmit: 'transmit'
-};
-
-/**
- * Some commands are converted to unicode (to provide arrow characters) for specific shortkeys
- *
- * This maps those unicode values, converted to a string, to the correct root command
- *
- * @property UNICODE_COMMANDS
- * @type {Object}
- * @final
- */
-const UNICODE_COMMANDS = {
-    '\\u2B50': 'land'
-};
+import _findKey from 'lodash/findKey';
 
+// FIXME: These keys should be upper-cased, but waiting for now to keep things working
```

Comment:
why should they be upper-cased?  since these map directly to method names, I'd say it makes sense for them to be camelCase.  We should probably have a comment as to _why_ they are camelCased, though.

Reply:
Kinda just because it's been the pattern. If it doesn't bother you, I'll leave it because frankly I've grown tired of this PR anyway. 😉

Reply:
turns out there was already a comment to this effect, so i simply removed the fixme.